packages/tldraw/src/lib/ui/context/TldrawUiContextProvider.tsx

```diff
@@ -52,7 +52,7 @@ export interface TLUiContextProviderProps {
 }
 
 /** @public @react */
-export function TldrawUiContextProvider({
+export const TldrawUiContextProvider = track(function TldrawUiContextProvider({
```

Comment:
What are we capturing here?

Reply:
Similar to the language menu hotfix that went out, it's so the lang menu is reactive: https://github.com/tldraw/tldraw/pull/4685